### PR TITLE
Add Monthly Beatmapping Contest: February news post

### PR DESCRIPTION
**Pull request by pishifat**
- [x] requires #3139 
- [x] replace "top XX" with actual value

### CODE REVIEW

File: news/2020-02-03-monthly-beatmapping-contest-february.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+If you're curious about the details behind any of these, dive into the [Monthly Beatmapping Contest wiki article](/wiki/Contests/Monthly_Beatmapping_Contest). It answers every question you could ask. Probably.
+
+## Beatmapping
+
+### <a id="limitation"></a>Limitation
+
+Each month's contest has its own mapping limitation that participants must design their maps around. For example, last month's limitation was "straight slider shapes only", so sliders in all beatmap submissions were... straight lines. Pretty simple, right?
+
+This month's limitation: **star rating cannot exceed 4.**
```
Comment (Coppertine) [inline news/2020-02-03-monthly-beatmapping-contest-february.md]: can spesify by saying "star rating cannot exceed 4.00" as people can see "4." as 4.99* instead

Comment (Pachiru) [reply news/2020-02-03-monthly-beatmapping-contest-february.md]: It's just a period, I don't think people would misunderstand that :/

Comment (Coppertine) [reply news/2020-02-03-monthly-beatmapping-contest-february.md]: > [Random mapper] Today at 9:23 PM
> 4.00 stars or 4.99 stars ?
it's more of a thing where you either include the section of the 4 to 4.99 or make it max to 4.00

Comment (venix12) [reply news/2020-02-03-monthly-beatmapping-contest-february.md]: yes i think 4.00* would be more clear

File: news/2020-02-03-monthly-beatmapping-contest-february.md [hunk 2]
```diff
@@ -51,39 +50,39 @@ This month's artist is **Voicians**, and the following songs are viable for cont
 
 #### Closer (172 BPM)
 
-Beatmap template download: [Full version](https://assets.ppy.sh/artists/59/osz/Voicians%20-%20Closer.osz) (4:00) / [Short version](https://assets.ppy.sh/contests/89/files/Voicians_-_Closer.osz) (2:20)
+Beatmap template download: [Full version](https://assets.ppy.sh/artists/59/osz/Voicians%20-%20Closer.osz) (4:00) / [Short version](https://assets.ppy.sh/contests/89/files/Voicians_-_Closer.osz?2020-1) (2:20)
```
Comment (cl8n) [inline news/2020-02-03-monthly-beatmapping-contest-february.md]: why was this necessary? there doesn't seem to be any caching issue

Comment (Coppertine) [reply news/2020-02-03-monthly-beatmapping-contest-february.md]: I think it's more for versioning, making sure you get the right version of the osz.

Comment (pishifat) [reply news/2020-02-03-monthly-beatmapping-contest-february.md]: was told to add it by guy who uploaded files to assets.ppy.sh

Comment (cl8n) [reply news/2020-02-03-monthly-beatmapping-contest-february.md]: well it doesn't really harm anything either so good enough for me.

File: news/2020-02-03-monthly-beatmapping-contest-february.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+- replace "top XX" with actual value
+-->
+
+New month? New mapping contest! Vote on last month's submissions and compete to become the next Elite Mapper!
+
+![](/wiki/shared/news/banners/monthly-beatmapping-contest.png)
+
+The Monthly Beatmapping Contest (MBC) is back after a multi-year hiatus. Check out [last month's contest announcement](https://osu.ppy.sh/home/news/2020-01-05-monthly-beatmapping-contests-return) if you need to catch up.
+
+January's contest received a *massive* turnout of over 100 valid submissions! A group of experienced beatmappers narrowed them down to the top XX — the best of the best — so we encourage you to download those and vote for your favorite.
```
Comment (cl8n) [inline news/2020-02-03-monthly-beatmapping-contest-february.md]:
```suggestion
January's contest received a *massive* turnout of over 100 valid submissions! A group of experienced beatmappers narrowed them down to the top XX — the best of the best — so we encourage you to download those and vote for your favourite.
```
cuz we are definitely not americans

File: news/2020-02-03-monthly-beatmapping-contest-february.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+
+**Vote for last month's contest [here](https://osu.ppy.sh/community/contests/84)!** Results will be published in 2 weeks.
+
+Now that housekeeping is out of the way, let's talk about the contest this month!
```
Comment (cl8n) [inline news/2020-02-03-monthly-beatmapping-contest-february.md]:
```suggestion
Now that housekeeping is out of the way, let's talk about this month's contest!
```

File: news/2020-02-03-monthly-beatmapping-contest-february.md [hunk 5]
```diff
@@ -10,7 +10,7 @@ date: 2020-02-03 11:00:00 +0000
 
 New month? New mapping contest! Vote on last month's submissions and compete to become the next Elite Mapper!
 
-![monthly beatmapping contest banner](/wiki/shared/news/banners/monthly-beatmapping-contest.png)
+![](/wiki/shared/news/banners/monthly-beatmapping-contest.png)
```
Comment (cl8n) [inline news/2020-02-03-monthly-beatmapping-contest-february.md]: the reason for this is that alt text is pretty much useless for banners, if the image didn't load there would still be room for the banner and the title overlaying it

File: news/2020-02-03-monthly-beatmapping-contest-february.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+
+<!-- todo
+- replace "top XX" with actual value
+-->
+
+New month? New mapping contest! Vote on last month's submissions and compete to become the next Elite Mapper!
+
+![](/wiki/shared/news/banners/monthly-beatmapping-contest.png)
+
+The Monthly Beatmapping Contest (MBC) is back after a multi-year hiatus. Check out [last month's contest announcement](https://osu.ppy.sh/home/news/2020-01-05-monthly-beatmapping-contests-return) if you need to catch up.
```
Comment (cl8n) [inline news/2020-02-03-monthly-beatmapping-contest-february.md]:
```suggestion
The Monthly Beatmapping Contest (*MBC*) is back after a multi-year hiatus. Check out [last month's contest announcement](https://osu.ppy.sh/home/news/2020-01-05-monthly-beatmapping-contests-return) if you need to catch up.
```
missed that
